### PR TITLE
chore: add UI changes to changelog for 0.12 release

### PR DESCRIPTION
**Pull request by cameronperera**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
 * metrics: Adds accepted connections and closed connections counters to keep track
   downstream connections for worker and controller servers.
   ([PR](https://github.com/hashicorp/boundary/pull/2668))
+* ui: Upgrade admin UI to Ember to 4.4 ([PR](https://github.com/hashicorp/boundary-ui/pull/1523)).
```
Comment (calcaide) [inline CHANGELOG.md:18]: Not sure what we really want to say here, but "admin ui to Ember to 4.4" sounds redundant to me.
How about?
- "Upgrade Admin UI to Ember 4.4" or "Upgrade Admin UI from Ember 3.28.x to Ember 4.4". Not sure how descriptive we want to be.

**Review by calcaide — CHANGES_REQUESTED** [review]
LGTM! left a comment

File: CHANGELOG.md (hunk 1 above)
Comment (cameronperera) [reply CHANGELOG.md:18]: Yeah that does read odd. I am not sure what I was thinking when I typed it 😆 
Thanks for the suggestions. I will update now.

**Review by ZedLi — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -23,6 +25,8 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
   plugin to exit ([PR](https://github.com/hashicorp/boundary/pull/2677))
 * data warehouse: Fix bug that caused credential dimensions to not get
     associated with session facts ([PR](https://github.com/hashicorp/boundary/pull/2787)).
+* ui: Add support for managed groups in add-principals list ([PR](https://github.com/hashicorp/boundary-ui/pull/1548)).
```
[the remark this answers is not in the document]
Comment (cameronperera) [reply CHANGELOG.md:28]: Good question. I assumed this was meant as a fix. I will check and update if needed.

File: CHANGELOG.md [hunk 3]
```diff
@@ -15,6 +15,8 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
 * metrics: Adds accepted connections and closed connections counters to keep track
   downstream connections for worker and controller servers.
   ([PR](https://github.com/hashicorp/boundary/pull/2668))
+* ui: Upgrade Admin UI to Ember 4.4 ([PR](https://github.com/hashicorp/boundary-ui/pull/1523)).
+* ui: Added support for JSON credentials in admin UI ([PR](https://github.com/hashicorp/boundary-ui/pull/1498)).
```
Comment (randallmorey) [inline CHANGELOG.md:19]: Past or present tense?

File: CHANGELOG.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (randallmorey) [reply CHANGELOG.md:28]: This is more a feature than a bug.

**Review by randallmorey — CHANGES_REQUESTED** [review]

**Review by randallmorey — CHANGES_REQUESTED** [review]
@cameronperera it looks like the content is wrapped at around 80 with newlines.

File: CHANGELOG.md (hunk 3 above)
Comment (cameronperera) [reply CHANGELOG.md]: Fixed.

**Comment by cameronperera** [comment]
Closing due to new process for changelog entries of features that are being released from the BE and FE teams at the same time.

**Pull request closed without merging**